client/src/components/Donation/DonateForm.tsx

```diff
@@ -334,8 +335,9 @@ class DonateForm extends Component<DonateFormProps, DonateFormComponentState> {
             isPaypalLoading={loading.paypal}
             isSignedIn={isSignedIn}
             onDonationStateChange={this.onDonationStateChange}
-            theme={defaultTheme ? defaultTheme : theme}
+            theme={priorityTheme}
```

Comment:
I don't think the defaultTheme should take precedence over the theme, is that the intention?  Normally the default should be the fallback that gets overridden.

Reply:
So on components such as the certification, we pass the defaulthTeme props down to the DonateForm. As a result we show the default theme for that page regardless of the user's theme preferences.

Reply:
Sure, it makes sense. TBH I didn't want to touch on themes on this pr, but it was on the line above what I was working on. :)

Reply:
Yeah, understood. I just figured it was worth a quick discussion.